test/k8sT/Policies.go

```diff
@@ -2354,6 +2338,228 @@ var _ = SkipDescribeIf(func() bool {
 	})
 })
 
+// Create a new Describe block because we need to run some tests in GKE. For
+// example, the kube-apiserver policy matching feature needs coverage on GKE as
+// there are two cases for that feature:
+//   * kube-apiserver running within the cluster (Vagrant VMs)
+//   * kube-apiserver running outside of the cluster (GKE)
+var _ = SkipDescribeIf(helpers.DoesNotRunOn419OrLaterKernel,
+	"K8sPolicyTestExtended", func() {
+		var (
+			kubectl *helpers.Kubectl
+
+			// these are set in BeforeAll()
+			ciliumFilename string
+			daemonCfg      map[string]string
+		)
+
+		BeforeAll(func() {
+			kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+			daemonCfg = map[string]string{
+				"tls.secretsBackend": "k8s",
+				"debug.verbose":      "flow",
+				"hubble.enabled":     "true",
+			}
+			ciliumFilename = helpers.TimestampFilename("cilium.yaml")
+			DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, daemonCfg)
```

Comment:
Why do we need to deploy Cilium here? Wouldn't deploying in the `Context` block be enough?

Reply:
Good point, there's no reason to. It would just bloat the test time.